app/helpers/handlers.js

```diff
@@ -56,15 +56,15 @@ const handlePotentialAmount = (request, maybeEligibleContent, url) => {
     return {
       ...maybeEligibleContent,
       detailsText: {
-        summaryText: 'How is the solar PV system grant funding calculated?',
+        ...maybeEligibleContent.detailsText,
```

Comment:
Should this file be part of the PR? Seems Solar PV removal related.

Reply:
This is not changing anything - it is improving readability. We can have a call about it if you want. Let me know.

Reply:
Yeah I take the point. I just wonder with the lack of test coverage of the application, whether any non-storage percentage change should be left out to limit the test requirement. This could go in the Solar PV removal later?

Reply:
Sure we can discuss this but haven't we said we want to make small improvements to the code around the requested change? If we are not doing small tech debt fixes as BAU and we do not have separate cards tackling tech debt how are we going to fix this? WDYT?

Reply:
Another thing is this grant is not live yet so why not making improvements along requested changes as we go?

Reply:
Yeah all true, and we do have reasonable acceptance test coverage of adding-value.